human.descriptions.py

```diff
@@ -1842,6 +2063,7 @@ def main():
             'ca': { 'male': "historiador de l'art ~", 'female': "historiadora de l'art ~" }, 
             'en': { 'male': '~ art historian', 'female': '~ art historian' }, 
             'es': { 'male': 'historiador del arte ~', 'female': 'historiadora del arte ~' }, 
+            'fr': { 'male': 'historien de l’art ~', 'female': 'historienne de l’art ~' }, 
```

Comment:
it is only a matter of personal preference. I used "’" to avoid to replace "'" by """